src/detours.h

```diff
@@ -543,8 +543,8 @@ PVOID WINAPI DetourSetSystemRegionUpperBound(_In_ PVOID pSystemRegionUpperBound)
 
 ////////////////////////////////////////////////////////////// Code Functions.
 //
-PVOID WINAPI DetourFindFunction(_In_ LPCSTR pszModule,
-                                _In_ LPCSTR pszFunction);
+PVOID WINAPI DetourFindFunction(_In_ PCSTR pszModule,
```

Comment:
Could you change the definition of the function to match the declaration instead?  You altered the declaration.   LPCSTR has slightly better information for static analysis.    From documentation:
```
LPCSTR

This type is declared in WinNT.h as follows:
typedef __nullterminated CONST CHAR *LPCSTR;
```
versus:
```
PCSTR

This type is declared in WinNT.h as follows:
typedef CONST CHAR *PCSTR;
```
LPCSTR has a __nullterminated SAL annotation on it.

Reply:
That's a good point, I corrected it in the latest commits.